swap/src/bitcoin/wallet.rs

```diff
@@ -111,19 +111,18 @@ impl Wallet {
         Ok(())
     }
 
-    pub async fn build_tx_lock_psbt(
+    pub async fn send_to_address(
```

Comment:
🔧 I find this rename confusing, because `send_to_address` for me implies publishing the transaction to the network. However, this function just constructs a PSBT and has nothing to do with publication as far as I can see. How about `build_psbt`?

Reply:
I see, yes the name could use improvement to say it only builds a transaction and doesn't send it. Will give it some more thought!